Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/SpeechInputHandler.cs

```diff
@@ -26,6 +28,29 @@ public class SpeechInputHandler : BaseInputHandler, IMixedRealitySpeechHandler
         [Tooltip("Keywords are persistent across all scenes.  This Speech Input Handler instance will not be destroyed when loading a new scene.")]
         private bool persistentKeywords = false;
 
+        [SerializeField]
+        [Tooltip("Assign SpeechConfirmationTooltip.prefab here to display confirmation label. Optional.")]
+        private GameObject speechConfirmationTooltipPrefab = null;
+
+        /// <summary>
+        /// Tooltip prefab used to display confirmation label. Optional.
+        /// </summary>
+        public GameObject SpeechConfirmationTooltipPrefab
+        {
+            get { return speechConfirmationTooltipPrefab; }
+            set
+            {
+                if (speechConfirmationTooltipPrefab != value)
+                {
+                    speechConfirmationTooltipPrefab = value;
+                }
+            }
+        }
+
+        [SerializeField]
+        [Tooltip("Instance of the Speech confirmation tooltip prefab.")]
+        private GameObject speechConfirmationTooltipPrefabInstance = null;
```

Comment:
Since the intention is for this instance to be dynamically instantiated (and then self-destruct) this probably shouldn't be serialized. It might be confusing.

Reply:
Right. Removed the [SerializeField]